### ISSUE TITLE
Show count by default

### ISSUE THREAD

**Issue by Jonofat** [wp-graphql/wp-graphql#1863]
I know this isn't an issue but I looked in Discussions->ideas and there isn't much activity so not sure if this would ever get read. Would it possible that for every query there could be the data you want returned as well as a count of the number of records displayed. This would be so useful. Like now I want to show number of posts for a user

```
query MyQuery {
  posts(where: {author: 89}) {
    nodes {
      title
    }
  }
}
```

It would be awesome if this returned a count that I could use.

**Comment by jasonbahl** [comment]
@Jonofat ya, this is something that's been discussed for some time. SQL calculations for total counts can get expensive and slow, so it's often [best practice](https://10up.github.io/Engineering-Best-Practices/php/#performance) to avoid this. 

I've left it out as default behavior to prevent these slow-downs. Of course, there is utility to exposing totals, and on small sites, it's usually not even a performance issue. 

So, there's 2 thoughts I have. Expose it by default, and allow servers to opt-out of it (big sites, for example might want to remove clients access to totaling a million rows). Or leave it out by default, and let servers opt-in to exposing it on some/all connections. 

In any case, you should be able to implement this using hooks/filters today if it's needed for your specific projects. 

There's more discussion here as well: https://github.com/wp-graphql/wp-graphql/issues/723

**Comment by Jonofat** [comment]
Thanks so much for the response @jasonbahl I will look into the hooks/filters.

**Comment by justlevine** [comment]
leaving closed in favor of #1888
